appinfo/routes.php

```diff
@@ -55,6 +55,7 @@
 		['name' => 'poll#list', 'url' => '/polls/list/', 'verb' => 'GET'],
 		['name' => 'poll#get', 'url' => '/polls/get/{pollId}', 'verb' => 'GET'],
 		['name' => 'poll#delete', 'url' => '/polls/delete/{pollId}', 'verb' => 'GET'],
+		['name' => 'poll#deletePermanently', 'url' => '/polls/delete/permanent/{pollId}', 'verb' => 'GET'],
```

Comment:
Any reason for not using the `DELETE` verb?
```suggestion
		['name' => 'poll#deletePermanently', 'url' => '/polls/delete/permanent/{pollId}', 'verb' => 'DELETE'],
```

Reply:
Itried it, but it didn't work. Thus I thought this isn't supported in NC

Reply:
I think it should 😁 a quick research gave me:

https://github.com/nextcloud/spreed/blob/master/appinfo/routes.php#L92-L100
```php
...
[
		'name' => 'Call#leaveCall',
		'url' => '/api/{apiVersion}/call/{token}',
		'verb' => 'DELETE',
		...
]
...
```
https://github.com/nextcloud/spreed/blob/master/src/services/participantsService.js#L75

```javascript
...
const leaveConversationSync = function(token) {
	axios.delete(generateOcsUrl('apps/spreed/api/v1/room', 2) + token + '/participants/active')
}
...
```

Reply:
For some strange reason it worked fine this time 😕 Maybe I did something wrong last time. Will push a fix when at my dev machine.

Reply:
FYI. Building works well on windows machines with some tools. Only signing is an issue.